_pages/dev/features/asm.md

```diff
@@ -84,76 +79,61 @@ UserGroup;asagentgroup;;;;;;;;
 
 ```text
 corsfilter.assistedservicewebservices.allowedOrigins=https://my-new-host:4200
 ```
 
-For development purposes only, the value can be a wildcard:
+For development purposes only, you can set the value to a wildcard (`*`), as shown in the following example:
 
 ```text
 corsfilter.assistedservicewebservices.allowedOrigins=*
 ```
 
-Bear in mind this wildcard configuration is flexible for development environments but it is unsecured. A more restrictive configuration is required for production use.
-
-## Invoke the ASM UI in the storefront
-
-To invoke the ASM UI in the storefront, add the `?asm=true` suffix to the url.
-For example, with the sample store, you can invoke the ASM UI on the home page with this url.
-
-```text
-https://{hostname}/electronics-spa/en/USD/?asm=true
-```
-
-## Update custom services to support ASM.
+**Note:** This wildcard configuration is flexible for development environments, but it is not secure. A more restrictive configuration is required for production use.
 
-If you use a custom service that extends one of these classes:
+## Updating Custom Services to Support ASM
 
-- UserService
-- UserAddressService
-- UserConsentService
-- UserOrderService
-- UserPaymentService
+If you use a custom service that extends one of the following classes, you need to update the relevant constructor to support ASM in your storefront:
 
-You need to update their constructor in order to support ASM in your storefront.
+- `UserService`
+- `UserAddressService`
+- `UserConsentService`
+- `UserOrderService`
+- `UserPaymentService`
 
-If during customer emulation the storefront displays the error message: `Cannot find user with propertyValue 'current'`, it is very likely that you use custom services that need to be updated to support ASM.
+During customer emulation, the storefront may display the following error message: `Cannot find user with propertyValue 'current'`. If you see this message, it is very likely that you have a custom service that needs to be updated to support ASM.
 
-The update consists of adding a dependency to AuthService in the constructor and pass it down to super(). The constructors that don't have AuthService are now deprecated.
+You can update your custom service by adding a dependency to the `AuthService` in the constructor, and passing it down to `super()`. Constructors that do not have `AuthService` are now deprecated.
 
-Beyond the constructor update, you may need to update your custom functions to support ASM as well. See the section called `How to write ASM compatible code` for details.
+In addition to updating the constructor, you may need to update your custom functions to support ASM as well. For more information, see [Writing ASM-Compatible Code](#writing-asm-compatible-code).
 
-### Update UserService Subclasses
+### Updating Subclasses
 
-#### Custom service
+If you use a custom service that extends the `UserService` class, your custom service may look like the following example:
```

Comment:
I think all of the content that explains how to write ASM compatible code needs a technical review.  I there is a chance this has changed since the session management refactoring.   Since this seems out of scope of the present update wee could create a follow-up ticket and address it separately.

Reply:
Yes, this is a good idea -- thank you for suggestion it. Follow up ticket #1341 has been created and assigned.